### DISCUSSION TITLE
Need enterprise plan

### DISCUSSION THREAD

**Discussion by poojakumari367** [community/community#110957] [Enterprise]
i want to purchase enterpise plan , need deatils for that please help me on that

**Comment by austings** [comment]
The GitHub Enterprise plan provides additional capabilities compared to the standard GitHub.com offering, such as:
Enhanced security features, tools for enterprise administration (managing users), and recovery options to ensure uptime & Dedicated support. Here is the link to the plan
https://github.com/organizations/enterprise_plan
